### DISCUSSION TITLE
briefcase create iOS Xcode==14.2 macOS Monterey==12.7.6

### DISCUSSION THREAD

**Discussion by jlij** [beeware/toga#2821] [Q&A] [answered]
An Xcode install appears to exist, but Briefcase was unable to
determine the current Xcode version. Running:

    $ xcodebuild -version

should return the current Xcode version, but it raised an error.

You may need to re-install Xcode. Re-run Briefcase once that
installation is complete.
[briefcase.2024_09_07-18_35_02.create.log](https://github.com/user-attachments/files/16917659/briefcase.2024_09_07-18_35_02.create.log)

**Comment by freakboy3742** [comment, marked as answer]
From the logs, it appears that you've installed Xcode.app in your `~/Downloads` folder, and that is the currently selected Xcode. You'll get the full error message if you run `xcodebuild -version` from a command line prompt - that will also provide the commands to fix the problem, which will depend on where you have installed Xcode.
